components/proof-of-yield/TableData.tsx

```diff
@@ -0,0 +1,30 @@
+import { PropsWithChildren } from "react";
+import { twMerge } from "tailwind-merge";
+
+interface TableDataProps {
+  align?: "left" | "right" | "center" | "justify" | "char";
+  className?: string;
+  width?: string;
+}
+
+const TableData = ({
+  align,
+  className,
+  width,
+  children,
+}: PropsWithChildren<TableDataProps>) => {
+  return (
+    <td
+      align={`${align || "right"}`}
```

Comment:
For consistency, might make sense to just stick with classnames here, text-right/text-left/text-center?

The <td> align attribute is not supported by HTML 5, its kind of obsolete/deprecated